src/main/resources/sql/db.changelog-1.0.sql

```diff
@@ -0,0 +1,41 @@
+--liquibase formatted sql
+
+--changeset aziz:1
+CREATE TABLE vehicle (
+    id BIGSERIAL PRIMARY KEY,
+    brand TEXT NOT NULL,
+    model TEXT NOT NULL,
+    production_year INTEGER NOT NULL,
+    kilometres DOUBLE PRECISION NOT NULL,
+    city TEXT NOT NULL,
+    price NUMERIC(10, 3) NOT NULL,
+    created_date TIMESTAMPTZ,
+    last_modified_date TIMESTAMPTZ
+);
+--rollback drop table vehicle;
+
+--changeset aziz:2
+CREATE TABLE picture (
+    id BIGSERIAL PRIMARY KEY,
+    url TEXT NOT NULL
+);
+--rollback drop table picture;
+
+--changeset aziz:3
+CREATE TABLE IF NOT EXISTS vehicle_pictures (
```

Comment:
You've created a many to many relationship. You only need a one to many relationship